### PR TITLE
improve smoke test docs

### PR DESCRIPTION
**Pull request by jackkav**
Trying to make it easier to follow the smoke test docs

### CODE REVIEW

File: packages/insomnia-smoke-test/README.md [hunk 1]
```diff
@@ -6,45 +6,60 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
 ```shell
-npm run app-start-playwright                    # Run Insomnia tests
-PWDEBUG=1 npm run test:smoke:dev                # Write Insomnia tests with the playwrite recorder
-DEBUG=pw:browser,pw:api npm run test:smoke:dev  # Run Insomnia tests, with verbose output
+# Run tests
+npm run test:smoke:dev
+# Debug tests with verbose output
+DEBUG=pw:browser,pw:api npm run test:smoke:dev
+# Step through tests with playwright inspector
+PWDEBUG=1 npm run test:smoke:dev
```
Comment (dimitropoulos) [inline packages/insomnia-smoke-test/README.md:37]: the way these are presented, it's always confusing at first when I look at them because it's like it's suggesting to me to run them all at once.  if they were split into separate codeblocks it might read easier and would allow you to comment outside of the script

Comment (dimitropoulos) [reply packages/insomnia-smoke-test/README.md:37]: Run tests

```shell
npm run test:smoke:dev
```

Debug tests with verbose output

```shell
DEBUG=pw:browser,pw:api npm run test:smoke:dev
```

Step through tests with playwright inspector

```shell
PWDEBUG=1 npm run test:smoke:dev
```

Comment (jackkav) [reply packages/insomnia-smoke-test/README.md:37]: I started with that but saw inconsistencies elsewhere but I don't mind changing it back

File: packages/insomnia-smoke-test/README.md [hunk 2]
```diff
@@ -6,45 +6,60 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+### Development method
 
-From the root of this repository:
+In one terminal run the watcher
 
 ```shell
-npm run bootstrap                   # Install packages and compile inso
-npm run app-build:smoke             # Compile Insomnia
-npm run inso-package                # Package the Inso CLI binaries
+npm run app-start-playwright    # Run watcher
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md]: Watcher would no longer be `app-start-playwright` after this PR, but `watch:app`
```suggestion
npm run watch:app   # Run watcher
```

File: packages/insomnia-smoke-test/README.md (hunk 1 above)
Comment (dimitropoulos) [reply packages/insomnia-smoke-test/README.md:37]: whatever you want to do is cool with me.

File: packages/insomnia-smoke-test/README.md [hunk 3]
```diff
@@ -6,45 +6,60 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+DEBUG=pw:browser,pw:api npm run test:smoke:dev
+# Step through tests with playwright inspector
+PWDEBUG=1 npm run test:smoke:dev
 ```
 
-Sometimes, you might need to run tests against a _packaged_ application. A packaged application is the final artifact which bundles all of the various resources together, and is created for distribution in the form of a `.dmg` or `.exe`, etc. Packaging takes longer to do and is only required for edge cases, so we typically run tests against a build. To run packaged tests, from the root:
+### Build and package methods
+
+Build = js bundle to load into an electron client
+Package = executable binary `.dmg` or `.exe`
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md]:
```suggestion
It's possible to run the smoke tests for:
- A `build`, the JS bundle that is loaded into an electron client;
- A `package`, the executable binary (e.g. `.dmg` or `.exe`);
```

File: packages/insomnia-smoke-test/README.md [hunk 4]
```diff
@@ -6,45 +6,60 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md:48]:
```suggestion
For `build`:
```

File: packages/insomnia-smoke-test/README.md [hunk 5]
```diff
@@ -6,45 +6,60 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+```shell
+npm run app-build:smoke             # Transpile js bundle
+npm run test:smoke:package          # Run tests
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md]: There's a typo here, you probably mean `build` instead of `package`:
```suggestion
npm run test:smoke:build          # Run tests
```

File: packages/insomnia-smoke-test/README.md [hunk 6]
```diff
@@ -6,45 +6,60 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+```
 
 ```shell
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md:56]:
```suggestion
For `package`:
```shell
```

**Review by filfreire — CHANGES_REQUESTED** [review]
@jackkav added some minor code suggestions.

**Review by dimitropoulos — APPROVED** [review]

File: packages/insomnia-smoke-test/README.md [hunk 7]
```diff
@@ -6,45 +6,62 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+- A `build`, the JS bundle that is loaded into an electron client;
+- A `package`, the executable binary (e.g. `.dmg` or `.exe`);
+For `build`:
+```shell
+npm run app-build:smoke             # Transpile js bundle
+npm run test:smoke:build          # Run tests
+```
 
+For `package`:
 ```shell
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md:50]: @jackkav  this should fix the lint errors:
```suggestion
It's possible to run the smoke tests for:

- A `build`, the JS bundle that is loaded into an electron client;
- A `package`, the executable binary (e.g. `.dmg` or `.exe`);

For `build`:

```shell
npm run app-build:smoke         # Transpile js bundle
npm run test:smoke:build          # Run tests
```

For `package`:

```shell
```

File: packages/insomnia-smoke-test/README.md [hunk 8]
```diff
@@ -6,45 +6,62 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+DEBUG=pw:browser,pw:api npm run test:smoke:dev
+# Step through tests with playwright inspector
+PWDEBUG=1 npm run test:smoke:dev
 ```
 
-Sometimes, you might need to run tests against a _packaged_ application. A packaged application is the final artifact which bundles all of the various resources together, and is created for distribution in the form of a `.dmg` or `.exe`, etc. Packaging takes longer to do and is only required for edge cases, so we typically run tests against a build. To run packaged tests, from the root:
+### Build and package methods
+
+It's possible to run the smoke tests for:
+- A `build`, the JS bundle that is loaded into an electron client;
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md]:
```suggestion

- A `build`, the JS bundle that is loaded into an electron client;
```

File: packages/insomnia-smoke-test/README.md [hunk 9]
```diff
@@ -6,45 +6,62 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+- A `package`, the executable binary (e.g. `.dmg` or `.exe`);
+For `build`:
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md:47]:
```suggestion

For `build`:

```

File: packages/insomnia-smoke-test/README.md [hunk 10]
```diff
@@ -6,45 +6,62 @@ Tests for the Electron app are written using [Playwright](https://github.com/mic
...
+```shell
+npm run app-build:smoke             # Transpile js bundle
+npm run test:smoke:build          # Run tests
+```
 
+For `package`:
```
Comment (filfreire) [inline packages/insomnia-smoke-test/README.md:54]:
```suggestion
For `package`:

```

**Review by filfreire — APPROVED** [review]
@jackkav these should fix the Github actions `lint` errors.
